src/app/forms/templates/test-records/section-templates/test/test-section.template.ts

```diff
@@ -123,6 +123,17 @@ export const TestSection: FormNode = {
               disabled: true,
               label: 'End time',
               viewType: FormNodeViewTypes.TIME
+            },
+            {
+              name: 'prohibitionIssued',
+              type: FormNodeTypes.CONTROL,
+              value: '',
+              label: 'Prohibition issued',
+              options: [
```

Comment:
Do you not need to specify the view and edit type for this control?

Reply:
You don't have to because I just want the default viewType and we're not handling edit yet